schedule.json

```diff
@@ -73,5 +73,10 @@
     "maintenance": "2021-10-19",
     "end": "2023-04-30",
     "codename": ""
+  },
+  "v15": {
+    "start": "2020-10-20",
```

Comment:
Should we put the 21st instead? To avoid two releases on the same day and make sure 14 will be LTS before the release of 15.

Reply:
Updated to the 21st.

<img width="980" alt="Screen Shot 2020-04-22 at 11 57 54 AM" src="https://raw.githubusercontent.com/nodejs/Release/5681ba5ae87329834f8fc9f1f04ca9747091348e/schedule.svg">